lib/ts/recipeImplementation.ts

```diff
@@ -308,6 +308,27 @@ export default function RecipeImplementation(recipeImplInput: {
                     return domain === normalisedsessionDomain;
                 }
             }
+        },
+
+        getClockSkewInMillis: function ({
+            accessTokenPayload
+        }: {
+            accessTokenPayload: any;
+            responseHeaders: Headers;
+        }): number {
+            logDebugMessage("getClockSkewInMillis: called");
+
+            const tokenIssuedAt = accessTokenPayload?.iat;
```

Comment:
Check if this works with st-node version 14. I think that is still supported by this SDK and the access token payload format changed in version 15. We may need to add a fallback here to make this work with old versions.